src/sign-in-webapp/README.md

```diff
@@ -69,12 +69,20 @@ Later the web app can make calls to the Microsoft 365 Graph API on behalf of sig
 
 ## Scaffold the web app by using the ASP.NET Core Razor project template
 
-1. Execute the following command to create the new web api project
+1. Install the latest `Microsoft.Identity.Web` templates from NuGet.org
 
    ```bash
-   dotnet new webapp --auth SingleOrg --aad-instance "https://login.microsoftonline.com/" --client-id ${AZURE_AD_APP_CLIENT_ID_WEBAPP} --tenant-id $(az account show --query tenantId --output tsv) --domain ${AZURE_AD_APP_DOMAIN} --called-api-url "https://graph.microsoft.com/v1.0/me"
+   dotnet new --install Microsoft.Identity.Web.ProjectTemplates::1.21.0
    ```
 
+1. Create a Microsoft identity platform (Azure AD v2.0) application. Execute the following command to create the web app project
+
+   ```bash
+   dotnet new webapp2 --auth SingleOrg --aad-instance "https://login.microsoftonline.com/" --client-id ${AZURE_AD_APP_CLIENT_ID_WEBAPP} --tenant-id $(az account show --query tenantId --output tsv) --domain ${AZURE_AD_APP_DOMAIN} --called-api-url "https://graph.microsoft.com/v1.0/me"
```

Comment:
We should include instructions for only for running the sample application, not creating it.

The tutorials on Docs will walk the customer through adding authentication to a base web app that was generated without using `--auth`.

Reply:
thanks for the feedback @mmacy, that ^ sounds reasonable. 

Done  | Addressed from [3959b](https://github.com/Azure-Samples/ms-identity-docs-code-csharp/pull/9/commits/3959bbe9d73c26a2f8db3a7f8777d11575ae3d69)

Reply:
Just marking this one resolved to remove the PR block.  @mmacy if there is any more feedback beyond the change that @ferantivero made a week ago, please let us know.